extensions/replication/queueProcessor/QueueProcessor.js

```diff
@@ -282,6 +320,7 @@ class QueueProcessor extends EventEmitter {
      * @param {boolean} [options.disableConsumer] - true to disable
      *   startup of consumer (for testing: one has to call
      *   processQueueEntry() explicitly)
+     * @param {boolean} [options.status] - if false, kafka consumer is disabled
```

Comment:
Same idea than said above, if carrying a status, better have an object containing the elements of the status (e.g. `status.paused`), otherwise better rename the parameter to `paused` (+ invert its meaning) to help understanding.